docs/DEPLOYMENT.md

```diff
@@ -0,0 +1,215 @@
+
+8. Under ECS, go to `Clusters` and create a new cluster with the name `plio-staging-cluster`. (skip this step if you've already created a Cluster when setting up the frontend repository)
+    1. Select `Networking only`. We will go with serverless deployment so that we don't worry about managing our own server instances.
+    2. Don't create a new VPC for your cluster. We'll use the VPC created in previous step in the next step of creating a service.
+    3. Click on the create button.
+    4. You will see the new cluster within the list of clusters.
+
+9.  Get into `plio-staging-cluster` and create a new service.
+
+       1. Set launch type to Fargate. We'll use serverless deployments for Plio.
+       2. Name the service as `plio-analytics-staging`.
+       3. Under task definition, select `plio-analytics-staging` and use latest revision.
+       4. Set the number of tasks to be one.
+       5. Service type to be `REPLICA`.
+       6. Minimum healthy percentage should be 100 and maximum percent to be 200. Minimum healthy percentage defines the percentage of tasks that should be running at all times. Maximum percent defines how many additional tasks the service can create in parallel to running tasks, before killing the running tasks.
+       7. Deployment type to be `rolling update`.
+       8. Keep other values as default.
+       9. Use the Cluster VPC and the subnet that you configured previously with Elastic IP.
+       10. Set Auto-assign public IP to `ENABLED`. Otherwise, it makes the task accessible only through VPC and not public.
+       11. Under load balancing, select the `Network Load Balancing` option and select the `plio-analytics-staging` load balancer.
+       12. Inside `Container to Load Balancer`, click on `Add to load balancer option` and select `plio-analytics-staging` in the target group.
+       13. For auto-scaling, go with `Do not adjust the service's desired count`  for staging.
+       14. Review and create the service.
+
+10. Create a `redis` cluster on AWS Elasticache.
+
+      1. Go to the [ElastiCache dashboard](https://console.aws.amazon.com/elasticache/home). Select the region and click on `Create` button.
+      2. Select `Redis` as your Cluster engine.
+      3. Select `Cluster mode disabled`. More details on cluster mode [here](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/Replication.Redis-RedisCluster.html).
+      4. Choose `Amazon Cloud` as your location.
+      5. Set the name of your cluster as `redis-plio-analytics-staging`.
+      6. Choose the latest `Engine version compatibility`.
+      7. Enter `6379` as the `Port`.
+      8. No need to change anything for `Parameter Group`.
+      9. Choose your `Node Type`. Plio uses `cache.t2.micro`. More details [here](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/nodes-select-size.html#CacheNodes.SelectSize).
+      10. For `Number of replicas`, enter `2`.
```

Comment:
REMINDER: delete and create a new cluster on ElastiCache

Reply:
Done.